README.md

```diff
@@ -65,8 +65,9 @@ Since this list strives for completeness rather than listing only the best, it's
 - [Adde Misc](https://github.com/Darcman99/Adde-Endless-Sky) - Any Adde content that doesn't fit in another or deserve it's own branch. This includes many new outfits and ships for various aspects of the vanilla game, even new missions here and there.
   - [Adde Aben](https://github.com/Darcman99/Adde-Endless-Sky/tree/Aben) - New aliens that are uncapable of spaceflight as far as the player is concerned, offers the player multiple missions in aiding the Aben which also benefit the player in turn.
   - [Adde Irm](https://github.com/Darcman99/Adde-Endless-Sky/tree/Irm) - New aliens to the north of human space that can only be described as technocratic isolationists. They will not begin or turn hostile towards the player under normal circumstances, but you would be one unlucky pilot if you were to find yourself on bad terms with the Irm.
+  - [Adde Korath Expansion](https://github.com/Darcman99/Adde-Endless-Sky/tree/Korath-Expansion) - This plugin is intended to experiment with the Korath by creating new content for them in the form of ships, outfits, and missions. Any content I consider to fit well with vanilla is put alongside the existing Korath while any less fitting content is kept out of the vanilla experience, only accessible through Omnis.
   - [Adde Omnis](https://github.com/Darcman99/Adde-Endless-Sky/tree/Omnis) - The most extensive all-content plugin to date. It includes many opportunities to test balance or ship builds against AI driven ships or fleets, all vanilla content and the content of many plugins available to use to put together different ship builds. Many more essential cheat tools are included into Omnis too!
-  - [Adde Quarg Expansion](https://github.com/Darcman99/Adde-Endless-Sky/tree/Quarg-Expansion) - This plugin was made to get an early glimpse into the possible future of the Quarg faction. A PR on hold is where a large part of this plugin came from, however a small part of it is mine and I intend it to fit in somewhat.
+  - [Adde Quarg Expansion](https://github.com/Darcman99/Adde-Endless-Sky/tree/Quarg-Expansion) - This plugin was made to get an early glimpse into the possible future of the Quarg faction. A PR on hold is where a large part of this plugin came from, however a small part of it is mine and I intend it to fit in somewhat. Any content I consider to fit well with vanilla is put alongside the existing Quarg while any less fitting content is kept out of the vanilla experience, only accessible through Omnis.
```

Comment:
Not sure about this this addition. It's a duplicate of the above addition, and these lines are already bordering on too long.